components/raftstore/src/store/entry_storage.rs

```diff
@@ -980,8 +1060,127 @@ impl<ER: RaftEngine> EntryStorage<ER> {
         self.last_term = last_term;
     }
 
+    pub fn entry_cache_warmup_state(&self) -> &Option<CacheWarmupState> {
+        &self.cache_warmup_state
+    }
+
+    pub fn entry_cache_warmup_state_mut(&mut self) -> &mut Option<CacheWarmupState> {
+        &mut self.cache_warmup_state
+    }
+
+    pub fn exit_entry_cache_warmup_state(&mut self) {
```

Comment:
maybe call clear_entry_cache_warmup_state is better?

Reply:
fixed